subsys/bluetooth/host/direction.c

```diff
@@ -767,3 +813,40 @@ int bt_df_conn_cte_rx_disable(struct bt_conn *conn)
 	return bt_df_set_conn_cte_rx_enable(conn, false, NULL);
 }
 #endif /* CONFIG_BT_DF_CONNECTION_CTE_RX */
+
+#if defined(CONFIG_BT_DF_CONNECTION_CTE_RSP)
+static int bt_df_set_conn_cte_rsp_enable(struct bt_conn *conn, bool enable)
+{
+	CHECKIF(!conn)
+	{
```

Comment:
minor: the { should be at the end of the first line (basically same coding style as for an if-statement)

Reply:
Thanks, I've corrected it here and in couple other functions also.

Reply:
Thanks, however please keep any coding style fixes to existing code in separate commits (they can still be part of the same PR though)

Reply:
Yeah, will do :-)